dist/index.html

```diff
@@ -6,5 +6,45 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Leaderboard list app</title>
   <script defer src="/leaderboard/index.bundle.js"></script></head>
-  <body></body>
+  <body>
+    <header>
+      <h1>Leaderboard</h1>
+    </header>
+    <main>
+      <div class="left">
+        <div class="header">
+          <h3>Recent scores</h3>
+          <a class="refresh">Refresh</a>
+        </div>
+        <ul class="list"></ul>
+      </div>
+      <div class="right">
+        <h3>Add your score</h3>
+        <form class="add" method="post" action="/">
+          <div>
+            <input
+              name="name"
+              id="name"
+              type="text"
+              maxlength="30"
+              placeholder="Your name"
+            />
+            <label for="name" hidden></label>
+            <input
+              name="score"
+              id="score"
+              type="number"
+              min="0"
```

Comment:
- [ ] Kindly, the HTML code should looks like in the [wireframe](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/images/leaderboard_wireframe.png), styling is not required for this project.

Your Implementation | Figma Design
:---:|:---:
![leadbad](https://user-images.githubusercontent.com/79058364/161972717-e010b65d-04cd-4a5b-99e4-047041d4d3e1.png)| ![lead](https://user-images.githubusercontent.com/79058364/161972836-02bc3c3a-cda8-4b56-a6cd-e1fb24bfe9ff.png)

Reply:
Yes please do it because you are not working on add scores yet by making a POST, GET request to the API, as per the requirement of this first step you just need to display the HTML without no API functionality